### DISCUSSION TITLE
Getting the data back into the RTE for edit/update

### DISCUSSION THREAD

**Discussion by devstc** [mantinedev/mantine#2161] [Q&A]
I've been able to use the RTE in a form as an input and store the data in a database as an HTML string, but I'm having difficulty getting the the data back into the RTE for edit/update purposes.  I'm trying to pass it through the 'value' prop,  I've been able to get [Object object] or 'undefined' to display in the editor, so it seems it's possible to get data back into the editor, but so far no luck getting it to display so it can be edited.  I've tried `JSON.stringify` and I'm using a package called 'html-react-parse' to parse the string from the database back into HTML.  Is there a way to do this?

**Comment by rtivital** [comment]
`@mantine/rte` package is built with quill. You can get editor ref – https://mantine.dev/others/rte/#get-editor-ref
Everything else should be covered in quill.js documentation – https://quilljs.com/docs/quickstart/
